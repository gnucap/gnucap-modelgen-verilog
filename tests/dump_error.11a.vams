
module resistor();
endmodule;
module resistor();
endmodule;

paramset const_res resistor;
	parameter real rr = 0.;
	.r = 1;
endparamset;


