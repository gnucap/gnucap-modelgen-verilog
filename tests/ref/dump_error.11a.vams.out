... r();
endmodule;

paramset const_res resistor;
                     ^ ? too many
